lib/DidKeyDriver.js

```diff
@@ -119,10 +147,16 @@ export class DidKeyDriver {
     if(!did) {
       throw new TypeError('"did" must be a string.');
     }
-
     const [didAuthority, keyIdFragment] = did.split('#');
     const fingerprint = didAuthority.substring('did:key:'.length);
-    const keyPair = await this.verificationSuite.fromFingerprint({fingerprint});
+    // get the multikey header from the fingerprint
+    const multikeyHeader = getMultiKeyHeader({fingerprint});
+
+    const keyTypeHandler = this.allowedKeyTypeHandlers.get(multikeyHeader);
+    if(!keyTypeHandler) {
+      throw new Error(`Unsupported "multikeyHeader".`);
+    }
+    const keyPair = await keyTypeHandler.fromFingerprint({fingerprint});
```

Comment:
We should make the newer multikey-based handlers the default way of doing things and just do `keyTypeHandler.from({publicKeyMultibase})` here. We should invert the adapter we have for the legacy stuff -- to instead convert those old "verificationSuite" things to provide `.from({publicKeyMultibase})` properly. So we should have a `createKeyTypeHandler` adapter helper instead of a `createVerificationSuite` adapter helper now.

Reply:
Addressed here [057f772](https://github.com/digitalbazaar/did-method-key/pull/55/commits/057f77243e5a679188611ecc28d66d4c892a9d4b)